kolibri/plugins/coach/assets/src/views/plan/assignments/IndividualLearnerSelector.vue

```diff
@@ -32,7 +32,7 @@
         <template #default="{ items }">
           <CoreTable
             :selectable="true"
-            :emptyMessage="$tr('noUsersMatch')"
+            :emptyMessage="noLearnersEnrolled$({ className: className })"
```

Comment:
[blocking] what happens here if you do have learners [learner_a, learner_b] enrolled in the class, and you search in the learners table for "learner_c"?

<img width="975" alt="Screenshot 2024-10-24 at 4 07 24 PM" src="https://github.com/user-attachments/assets/422a91e4-4300-4834-a99f-4695e222b1d0">

Reply:
I miss this. I think we will still see `noLearnerEnrolled` string. Which does not make sense. Should  we haddled that with` noUsersMatch' string?

Reply:
Yes, I think handling that with the `noUsersMatch` string, and using one or the other based on the condition, would be the way to go